spec/unit/mutant/ast/named_children_spec.rb

```diff
@@ -42,8 +42,18 @@
       end
 
       describe '#bar' do
-        it 'returns named child bar' do
-          expect(instance.bar).to be(node_bar)
+        context 'when node is present' do
+          it 'returns named child bar' do
+            expect(instance.bar).to be(node_bar)
+          end
+        end
+
+        context 'when node is NOT present' do
```

Comment:
:thumbsup: yeah I don't think it is a big deal either way, just probably good to be consistent